community_pool/errors.gno

```diff
@@ -0,0 +1,18 @@
+package community_pool
+
+import (
+	"errors"
+
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	errNoPermission      = errors.New("[GNOSWAP-COMMUNITY_POOL-001] no permission")
+	errNotRegistered     = errors.New("[GNOSWAP-COMMUNITY_POOL-002] not registered")
+	errAlreadyRegistered = errors.New("[GNOSWAP-COMMUNITY_POOL-003] already registered")
```

Comment:
Will these need to stay here if we remove the current grc20registry and replace it with /r/grc20reg?

Reply:
current impl of `grc20reg` doesn't handle errors like above, so for now it needs to be stay.